Api/User.py

```diff
@@ -47,7 +46,7 @@ def post(self):
                             "username":new_user.username,
                             "password":hashed_password
                           })
-        # import pdb;pdb.set_trace()
+        import pdb;pdb.set_trace()
```

Comment:
It's good to remove debuggers from your code and by doing this you will avoid performance and security issues as well.